Invoking command in '/home/rhody/ros2_dev/src/py_magnetometer_driver': PYTHONPATH=/home/rhody/ros2_dev/build/py_magnetometer_driver/prefix_override:/home/rhody/ros2_dev/install/py_magnetometer_driver/lib/python3.10/site-packages:${PYTHONPATH} /usr/bin/python3 setup.py egg_info --egg-base ../../build/py_magnetometer_driver build --build-base /home/rhody/ros2_dev/build/py_magnetometer_driver/build install --record /home/rhody/ros2_dev/build/py_magnetometer_driver/install.log --single-version-externally-managed
Invoked command in '/home/rhody/ros2_dev/src/py_magnetometer_driver' returned '0': PYTHONPATH=/home/rhody/ros2_dev/build/py_magnetometer_driver/prefix_override:/home/rhody/ros2_dev/install/py_magnetometer_driver/lib/python3.10/site-packages:${PYTHONPATH} /usr/bin/python3 setup.py egg_info --egg-base ../../build/py_magnetometer_driver build --build-base /home/rhody/ros2_dev/build/py_magnetometer_driver/build install --record /home/rhody/ros2_dev/build/py_magnetometer_driver/install.log --single-version-externally-managed
